src/fmelt.c

```diff
@@ -802,7 +803,8 @@ SEXP fmelt(SEXP DT, SEXP id, SEXP measure, SEXP varfactor, SEXP valfactor, SEXP
   }
   int protecti=0;
   dtnames = PROTECT(getAttrib(DT, R_NamesSymbol)); protecti++;
-  if (isNull(dtnames)) error(_("names(data) is NULL. Please report to data.table-help"));
+  if (isNull(dtnames))
+    error(_("names(data) is NULL. Please report to the data.table issue tracker"));
```

Comment:
actually this should probably just be

```suggestion
    internal_error("names(data) is NULL");
```